src/app/commands/command-display/command-display.component.ts

```diff
@@ -58,7 +67,7 @@ export class CommandDisplayComponent implements OnInit, AfterViewInit {
   }
 
   debrace(value: string) {
-    if (!value) {
+    if (!value || value.includes('`')) {  // funky <> handling
```

Comment:
This inline comment is kind of confusing to me. Is this if check handling for the literal characters `<>` or is that shorthand for a code block?

Reply:
I was typing up an explanation on why this behaved this way but then I found out that my reasoning was flawed - iterating on this again.

(in summary: this function replaces the literal characters `<>` with `&lt;` and `&gt;` but it wasn't supposed to inside code blocks (markdown "`"), so this naively ignored any value that had any code block in it)